packages/react/src/exampleTypes/StepSlider.tsx

```diff
@@ -0,0 +1,186 @@
+  return steps[value]
+}
+
+const StepSlider: FC<StepSliderProps> = ({
+  tree,
+  node,
+  formatter = toNumber,
+}) => {
+  const formatValue = (value: number): string => {
+    const realValue = getRealValue(steps, value)
+    const lastStepValue = steps[steps.length - 1]
+    if (realValue === lastStepValue) return `${formatter(realValue)}+`
+    return formatter(realValue).toString()
+  }
+
+  const steps = node.steps
+  const count = node.context.recordsCount ?? 0
+  const defaultRange = getDefaultRange(steps, node.range)
+  const [rangeValues, setRangeValues] = useState(defaultRange)
+
+  const onRangeUpdated = (range: number[]) => {
+    const realRange = [
+      getRealValue(steps, range[0]),
+      getRealValue(steps, range[1]),
+    ]
+    tree.mutate(node.path, { range: realRange })
+  }
+
+  const realFormattedMin = formatValue(rangeValues[0])
+  const realFormattedMax = formatValue(rangeValues[1])
+  const rangeLabel = `${realFormattedMin} -  ${realFormattedMax}`
+  const currentCount = formatter(count).toString()
+
+  return (
+    <Box>
+      <HStack paddingX="4px" justifyContent="space-between">
+        <Label value={rangeLabel} label="Range" />
+        <Label value={currentCount} label="Results" alignment="right" />
+      </HStack>
+      {/** This padding exists because of contexture native margins.
+       * When the slider thumb is encapsulated on the filter UI,
+       * part of it gets cut if no padding is applied. */}
+      <Box marginTop="6px" paddingX="11px">
+        <RangeSlider
+          aria-label={['min', 'max']}
+          min={steps[0]}
+          max={steps.length - 1}
+          minStepsBetweenThumbs={1}
+          onChange={setRangeValues}
+          onChangeEnd={onRangeUpdated}
+          defaultValue={defaultRange}
+          marginTop={2}
+        >
+          <RangeSliderTrack background={theme.color.gray[100]}>
+            <RangeSliderFilledTrack background={theme.color.blue[500]} />
+          </RangeSliderTrack>
+
+          <SliderThumb index={0} label={realFormattedMin} />
+          <SliderThumb index={1} label={realFormattedMax} />
+        </RangeSlider>
+      </Box>
+    </Box>
+  )
+}
+
+interface SliderThumbProps {
+  index: number
+  label: string
+}
+
+const SliderThumb: FC<SliderThumbProps> = ({ index, label }) => (
+  <Tooltip
+    label={label}
+    placement="top"
+    paddingX="12px"
+    boxShadow="0"
+    borderWidth="1px"
+    borderRadius={theme.radius.base.xs}
+    background={theme.color.gray[50]}
+    borderColor={theme.color.container.border.sm}
+    color={theme.color.text.body.primary}
+    closeDelay={1000}
+  >
+    <RangeSliderThumb
+      index={index}
+      borderWidth="2px"
+      background={theme.color.gray[50]}
+      borderColor={theme.color.blue[500]}
+      _hover={{ borderColor: theme.color.blue[700] }}
+      _focus={{ boxShadow: `0 0 0 3px ${theme.color.blue[100]}` }}
+      _active={{ borderColor: theme.color.blue[700] }}
+    />
+  </Tooltip>
+)
```

Comment:
Request for removal of FC and a swap. from `px` where feasible.

Reply:
_I will resolve this thread in favor of concentrating the debate on the thread above._